sources/packages/forms/src/form-definitions/sfaa2023-24.json

```diff
@@ -1,4 +1,5 @@
 {
+  "_id": "65e8d8dc940cd2f72a5f6304",
```

Comment:
please remove the _id in line 2 and owner in the line 11

Reply:
Thanks for catching that, should be removed again now.